web/assets/js/util/utils.js

```diff
@@ -126,12 +126,9 @@ class RandomUtil {
     }
 
     static randomUUID() {
-        let d = new Date().getTime();
-        return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
-            let r = (d + Math.random() * 16) % 16 | 0;
```

Comment:
Here it applied another variable `datetime` to the random generated value. Isn't it enough for making it more complex?
Another question: Why you did not use `crypto.randomUUID()` instead ?

Reply:
A shorter codes sometimes be better. It seems that it is not necessary to call a constructor to generate a UUID.

We cannot use ```crypto.randomUUID();``` because only runs when your domain is localhost, or when the request is via HTTPS. Otherwise, the browser will encounter ```crypto.randomUUID is not a function``` error. so the user has to activate SSL for the panel to create inbound.